### DISCUSSION TITLE
Welcome to GodotVolumetricRendering Discussions!

### DISCUSSION THREAD

**Discussion by riccardolops** [riccardolops/GodotVolumetricRendering#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mlavik1** [comment]
Thanks again for making this! Really nice to finally have a complete volume rendering plugin for Godot!  I'm looking forward to see more and more students and researchers using Godot for these kind of things now :) 

I used to be a big fan of Unity. I still think it's a great engine, but I don't like the direction the company has taken recently - but I *do* like the direction Godot is heading in, and the fact that it's 100% free and open source! Of course it's hard to compete with Unity's large selection of plugins, but I'm seeing more and more great Godot plugins being made, and this is definitely a great addition! So if you're hoping to make a positive change then you've done an amazing job :sunglasses: 

I'm hoping to make some contributions to this project when I have time :) And to make PRs for new features added to the Unity volume rendering plugin.

**Reply by riccardolops** [reply]
I need to thank you!🤗
I used your project in combination with the [MRTK](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity) from microsoft for a VR app for the visualization of medical imaging for a university project.
I took the initiative to reimplement the volumetric rendering in Godot since I share your sentiment about the direction Godot is heading.
Your implementation served as an excellent learning experience for me. Thanks again for your pioneering work!😎

**Reply by mlavik1** [reply]
That sounds cool! I'm really glad my plugin was useful :D 
Mentioning VR: Have you tested this Godot plugin in VR yet?

**Reply by riccardolops** [reply]
Not yet, but I plan on doing so. I'm pretty sure it will work fine in PCVR with a good GPU.
In the case of your implementation in unity I was able to also export it for android on the quest2 (with very poor performance 😅).
Should also be able to export it on android now that [Godot 4.x](https://github.com/godotengine/godot/pull/73257) mono supports it

**Reply by mlavik1** [reply]
Yeah I also tried it on Quest2 - and I had to downscale the dataset a bit to get an OK-ish performance 😅

**Comment by Siberik** [comment]
I am an aspiring C# programmer. My work consists mainly of simple WPF applications. I have a task to render DICOM files in college. I started using already learned WPF, which could not bear the load in any way. But when I saw your repository, I got very excited. After all, what you are doing can help many people in the future.

I will try to help and contribute as much as I can to this project.
